compass/landice/tests/ismip6_run_ais/slurm.08.run

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+#SBATCH  --job-name={{ EXP }}
+#SBATCH  --account=m1795
+#SBATCH  --nodes=5
+#SBATCH  --tasks-per-node=64
+#SBATCH  --exclusive
+#SBATCH  --time=15:00:00
+#SBATCH  --qos=regular
+#SBATCH  --constraint=haswell
```

Comment:
This template is currently designed specifically for Cori-Haswell (and for a specific account).  It would be better if we could make use of
https://github.com/MPAS-Dev/compass/tree/master/compass/job
to create a more general job script that could still meet your needs.  I suggest we try to do this in a follow-up PR that can also hopefully reduce the amount of manual work that goes into the config file.